tools/ota_e2e_tests/aws_ota_test/aws_ota_test_case.py

```diff
@@ -106,14 +96,14 @@ def runTest(self):
             if all(p == 0 for p in returnCodes):
                 testResult = self.run()
             else:
-                testResult = OtaTestResult(testName=self._name, result=OtaTestResult.ERROR, reason='Building or flashing failed. Please check the logs.')
+                testResult = OtaTestResult(testName=self._name, result=OtaTestResult.ERROR, summary='Building or flashing failed. Please check logs.')
         except Exception:
             logAppendage = traceback.format_exc()
             print(logAppendage)
-            testResult = OtaTestResult(testName=self._name, result=OtaTestResult.FAIL, reason=logAppendage)
-        
-        print('OTA E2E TEST RESULT: ' + testResult.result + '; because ' + testResult.reason)
-        
+            testResult = OtaTestResult(testName=self._name, result=OtaTestResult.ERROR, summary='Exception found during test execution. Please check logs.')
```

Comment:
I wanted the exception to appear on the jenkins test's page, but this is fine i suppose.

Reply:
In the TestResult.print(), it will report all test results information.